src/ui/base-page-storyEdit.js

```diff
@@ -26,6 +26,17 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
                         namespace: "uploadStateToCounters"
                     }
                 }
+            },
+            "blockManager.loadStoryContentIntoBlockUIs": {
+                record: {
+                    "{storyEditor}.story.model": {
+                        func: "{that}.createBlocksFromData",
```

Comment:
It's a bit ambiguous to use `{that}` here. I think we should update this to use `{blockManager}` or some other appropriate name that is more explicit.